packages/ui-react/src/components/Sidebar/Sidebar.tsx

```diff
@@ -21,14 +21,40 @@ const Sidebar: React.FC<SidebarProps> = ({ isOpen, onClose, children }) => {
   const htmlAttributes = { inert: !isOpen ? '' : undefined }; // inert is not yet officially supported in react. see: https://github.com/facebook/react/pull/24730
 
   useEffect(() => {
+    const scrollbarWidth = window.innerWidth - document.documentElement.clientWidth;
```

Comment:
We already have an util for calculating the scrollbar width (without triggering a repaint/reflow):

https://github.com/Videodock/ott-web-app/blob/3e3e2b14c4926e596fad46c2ea7cc99dbced05f0/packages/ui-react/src/utils/dom.ts#L5-L20

See https://gist.github.com/paulirish/5d52fb081b3570c81e3a